src/js/common/schemaform/SaveInProgressIntro.jsx

```diff
@@ -49,7 +49,7 @@ export default class SaveInProgressIntro extends React.Component {
         <div>
           <div className="usa-alert usa-alert-info schemaform-sip-alert">
             <div className="usa-alert-body">
-              You can save this form, and come back later to finish filling it out. To save your form in progress, please <button className="va-button-link" onClick={() => this.props.toggleLoginModal(true)}>Sign In</button>.
+              You can save this form, and come back later to finish filling it out. <button className="va-button-link" onClick={() => this.props.toggleLoginModal(true)}>Sign in to save your form in progress</button>.
```

Comment:
We might want to put a line break in here specifically, so we don't rely on wrapping.

Reply:
Great point, added.